src/client/components/Sidebar/CryptoChart.js

```diff
@@ -61,18 +69,27 @@ class CryptoChart extends React.Component {
       loading: true,
     });
     fetchCryptoPriceHistory(symbol).then((response) => {
```

Comment:
We should handle catch as well to prevent unhandled promise error.

Reply:
good point will add